src/OrbitGl/TimeGraph.cpp

```diff
@@ -643,6 +643,9 @@ void TimeGraph::UpdatePrimitives(PickingMode picking_mode) {
   track_manager_->SortTracks();
   track_manager_->UpdateMovingTrackSorting();
   track_manager_->UpdateTracks(&batcher_, min_tick, max_tick, picking_mode);
+  // Coordinates from CaptureWindows could need an update if we modified the vertical size of some
+  // track.
+  GetCanvas()->UpdateWorldTopLeftY();
```

Comment:
I'm a bit suprised this fixes the issue, and I'm not sure it will work in all cases. Problem is, this is happening after redraw, so it forces another redraw / update primitives if the height of any track changed. This happens in `UpdateWorldTopLeftY()`, but the line right below this resets `update_primitives_requested_` to false... this should mean that the next frame will call `Draw`, but not `UpdatePrimitives` on all tracks, shouldn't this lead to issues?

Second issue: Re-requesting `Draw` / `UpdatePrimitives` could lead to issues with picking. If this happens when picking is active, AFAIK there will be no picking-redraw...

I think the only clean solution would be completely decoupling position / size calculation from drawing.

Reply:
I'm merging without addressing this issue. Feel free to open a bug for checking what it is actually happening in the drawing process of CaptureWindows.